PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-resource-connector-lib/src/main/java/com/microsoft/azure/toolkit/intellij/connector/projectexplorer/ResourceNode.java

```diff
@@ -88,7 +92,25 @@ public void onClicked(Object event) {
     @Override
     @Nullable
     public IActionGroup getActionGroup() {
-        return Optional.ofNullable(getValue()).map(Node::getActions).orElse(null);
+        IActionGroup group = Optional.ofNullable(getValue()).map(Node::getActions).orElse(null);
+        final Object value = this.getValue().getValue();
```

Comment:
getValue is nullable, may use `Optional.ofNullable` or check null and return here